### PR TITLE
Fix storage types

### PR DESCRIPTION
**Pull request by virgil-serbanuta**

### CODE REVIEW

File: ulm-semantics/main/hooks/bytes.md [hunk 1]
```diff
@@ -345,6 +345,7 @@ module ULM-SEMANTICS-HOOKS-BYTES
 
     rule ulmBytesHash(ptrValue(_, u64(BytesId)))
         => ulmBytesHash(ulmBytesId(BytesId))
+    // TODO: use Keccack here.
```
Comment (Robertorosmaninho) [inline ulm-semantics/main/hooks/bytes.md:348]: Why is this needed?

**Review by Robertorosmaninho — APPROVED** [review]
LGTM

File: ulm-semantics/main/hooks/bytes.md (hunk 1 above)
Comment (virgil-serbanuta) [reply ulm-semantics/main/hooks/bytes.md:348]: Strictly speaking, we don't need keccack here. What we actually want is:
* A hash function that has a low collision probability
* Using all 256 bits of the hash.

It's just that we are already using keccack in other places so we wouldn't add new dependencies or complexity by using it.

Comment (virgil-serbanuta) [reply ulm-semantics/main/hooks/bytes.md]: I rewrote the TODO.
